com.unity.render-pipelines.high-definition/Documentation~/Custom-Pass-buffers-pyramids.md

```diff
@@ -0,0 +1,83 @@
+# Custom Pass buffers and pyramids
+
+This section describes how depth and color buffers work with custom passes in HDRP.
+
+When HDRP renders a scene, it stores some data to a render target or buffer, and uses this data to create a series of mipmaps. This series of mipmaps is called a pyramid. Custom passes in HDRP use depth pyramids and color pyramids. You can reference a pyramid in a shader.
+
+HDRP creates depth and color pyramids at given points in the render pipeline. For more information, see [Depth pyramid and color pyramid generation in HDRP](#Custom-Pass-pyramid-generation).
+
+<a name="Custom-Pass-buffers"></a>
+
+## Custom pass buffers
+
+HDRP custom passes use the following buffers:
+
+- [Depth buffer](#Custom-Pass-depth-buffer)
+- [Color buffer](#Custom-Pass-color-buffer)
+
+You can set each of these buffers to use:
+
+- The camera’s color or depth buffer.
+- A [custom buffer](Custom-Pass-Creating.md#using-a-custom-buffer-format) under your control.
+- No buffer to disable color or depth outputs.
+
+<a name="Custom-Pass-depth-buffer"></a>
+
+### Depth buffer
+
+During rendering, HDRP uses a depth buffer to perform depth tests. By default, only opaque objects write to the depth buffer up to and including the BeforePreRefraction [injection point](Custom-Pass-Injection-Points.md).
+
+However, you can also enable HDRP to write transparent objects to the depth buffer. To do this, go to [Surface options](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@10.4/manual/Lit-Shader.html) and either enable **Depth Write**, **Transparent Depth Prepass** or **Transparent Depth Postpass** depending on when you want them to write to the depth buffer**.**
+
+You cannot read the current depth buffer from a rendering shader.
```

Comment:
That's not entirely true, it's possible to read it if you bind it from C# in a script custom pass